pkg/logging/logging.go

```diff
@@ -27,3 +33,30 @@ const (
 	Mutator               = "mutator"
 	DebugLevel            = 2 // r.log.Debug(foo) == r.log.V(logging.DebugLevel).Info(foo)
 )
+
+func LogStatsEntries(client *constraintclient.Client, logger logr.Logger, entries []*instrumentation.StatsEntry, msg string) {
+	for _, se := range entries {
+		labelledLogger := logger
```

Comment:
We should just log the entry as the message, logger will render the JSON structure.

Trying to fold the nested object into log labels seems fraught.

We might have to workshop how to deal with if `se.Status` is a long list, but I don't think that will be the case?

Reply:
`se.Stats` could be a long list depending on the number of `stats` that the CF gives back.

Today, that's small-ish ~ 2 and we know that because we implemented it. In the future, the number of `Stats` could grow so maybe stuffing everything into a log line could be not as helpful.

> Trying to fold the nested object into log labels seems fraught.

I'm curious why that would be?

Reply:
> I'm curious why that would be?

The need to worry about collision (what happens if there is a label named "name"?), potential loss of associative context (if multiple queries are writing out logs at the same time, it's harder to unpack which log line belongs to which request).

> Today, that's small-ish ~ 2 and we know that because we implemented it. In the future, the number of Stats could grow so maybe stuffing everything into a log line could be not as helpful.

I'd rather take the risk of needing to figure out pagination in the future than worrying about  uniquifying labels.

Reply:
> (if multiple queries are writing out logs at the same time, it's harder to unpack which log line belongs to which request)

this's won me over 💯 ; I'll make the code changes.

Reply:
Logging stats with the underlying encoder as suggested above now.